### ISSUE TITLE
Video setup silent failure mode

### ISSUE THREAD

**Issue by EskelCz** [prebid/Prebid.js#10637]
## Type of issue

error handling

## Description

When setting up video format to hook up into a player, I have found a failure mode which isn't treated as a failure mode but I think it should be. When the mediaTypes.video.divId doesn't match the video player div id, the video ad is not shown, without any error or warning.

## Steps to reproduce
Change divId in adUnit configuration to some non-existent value

## Test page
[https://stackblitz.com/edit/prebid-video?file=index.html%3AL43](https://stackblitz.com/edit/prebid-video?file=index.html%3AL43)

### Expected results
Ideally a warning that the player was not found

### Actual results
The setup acts as if the ad wasn't supposed to be played at all

## Platform details
Prebid 8.17 with a renderAd fix from this PR [https://github.com/prebid/Prebid.js/pull/10567](https://github.com/prebid/Prebid.js/pull/10567) 

## Other information
Moved to the separate issue from here: https://github.com/prebid/Prebid.js/issues/10505

**Comment by EskelCz** [comment]
Additionally, when the divId is wrong in `video.providers` object, the player breaks completely, also without any error or explanation.
Demo: https://stackblitz.com/edit/prebid-video-2?file=index.html%3AL65-L65

**Comment by dgirardi** [comment]
For the OP: https://github.com/prebid/Prebid.js/pull/10664

For the second issue: in theory you should get a `videoSetupFailed` event, but from what I can tell there's no way to access it - event listeners can be attached only after init, but the init itself will fire the error event at a time when there cannot be any listeners. @karimMourra could you help? I'm not sure if I'm missing something.

**Comment by karimMourra** [comment]
@EskelCz thanks for reporting and including the demo pages! Those are very helpful!
 @dgirardi thanks for the fix on #10664 - The other issue  appears to be a race condition where the `setupFailure` is triggered before the event listeners are attached. I'll try to propose a fix next week.
